samples/application_development/external_lib/sample.yaml

```diff
@@ -2,6 +2,10 @@ sample:
   name: External Library
 tests:
   test:
-    platform_whitelist: qemu_x86
```

Comment:
maybe a line in the commit message on why this whitelist was removed.

Reply:
done